build-capa.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+set -e
+function subst_env_vars {
+    local ORIG_FILE="$1"
+    local T_FILE="$2"
+    cp "$ORIG_FILE" "$T_FILE"
+    declare -a subst=(
```

Comment:
Same her for overlay the deployment CR, Why not use kustomize to replace those fields. Both apply for CAPI & CAPA.